pyanaconda/modules/payloads/payload/live_image/installation.py

```diff
@@ -440,6 +440,26 @@ def run(self):
             msg = "Failed to install image: {}".format(e)
             raise PayloadInstallationError(msg) from None
 
+        if os.path.exists(os.path.join(self._mount_point, "boot/efi")):
+            # Handle /boot/efi separately due to FAT filesystem limitations
+            # FAT cannot support permissions, ownership, symlinks, hard links,
+            # xattrs, ACLs or modification times
+            args = [
+                "-rx",
+                "--stats",  # show statistics at end of process
+                "--info=flist2,name,progress2",  # show progress after each file
+                "--no-inc-recursive",  # force calculating total work in advance
+                "--exclude", "/boot/efi/loader/",
+                os.path.normpath(self._mount_point) + "/boot/efi/",
+                os.path.join(self._sysroot, "boot/efi")
+            ]
+
+            try:
+                execWithRedirect(cmd, args)
```

Comment:
Could you please add
```
self.report_progress(_("Installing the /boot/efi content..."))
```
so this is not completely silent operation for users?

Reply:
I tested this interactively, and it tasks <1second to rsync this, it would be worse I believe to get the flashing string change.

Reply:
I see. In that case you can ignore this I guess.